packages/eds-core-react/src/components/Chip/Chips.stories.tsx

```diff
@@ -13,19 +14,15 @@ Icon.add(icons)
 const Wrapper = styled.div`
   display: grid;
   grid-gap: 32px;
-  grid-template-columns: repeat(3, fit-content(100%));
+  grid-template-columns: repeat(4, fit-content(100%));
 `
 
 export default {
   title: 'Data Display/Chips',
   component: Chip,
   parameters: {
     docs: {
-      description: {
-        component: `Chips, also known as tags or badges, are
-        advanced badges that represent discrete information.
-        `,
-      },
+      page,
     },
   },
 } as Meta
```

Comment:
This should be replaced with `ComponentMeta<typeof Chip>` from `@storybook/react` as its the new recommended way for meta information.